### PR TITLE
Add 2 new workouts to the Go topic

### PR DESCRIPTION
**Pull request by Stefan-Stojanovic**
- [x] Extra types-i
- [x] Extra types-ii

### CODE REVIEW

**Comment by enki-hq** [comment]
### Insight

| Insight Name | Error Message |
|:-:|:-:|
| range | Insight slug `range` is not unique in the database. |
| arrays | Insight slug `arrays` is not unique in the database. |
| slices | No matching glossary entry found in the database or this pull request. |
| make-a-slice | No matching glossary entry found in the database or this pull request. |
### Workout

| Workout Name | Error Message |
|:-:|:-:|
| extra-types-i | One of the mentioned insights couldn't be found in this pull request or the database. |
### Changelog

| File Name | Error Message |
|:-:|:-:|
| Pull Request | Couldn't find a CHANGELOG entry for this pull request. |

> If you notice anything wrong with these rules contact andrei@enki.com.

**Comment by enki-hq** [comment]
### Insight

| Insight Name | Error Message |
|:-:|:-:|
| arrays | Insight slug `arrays` is not unique in the database. |
| range | Insight slug `range` is not unique in the database. |
| slices | No matching glossary entry found in the database or this pull request. |
| make-a-slice | No matching glossary entry found in the database or this pull request. |
### Workout

| Workout Name | Error Message |
|:-:|:-:|
| extra-types-i | One of the mentioned insights couldn't be found in this pull request or the database. |
### Changelog

| File Name | Error Message |
|:-:|:-:|
| Pull Request | Couldn't find a CHANGELOG entry for this pull request. |

> If you notice anything wrong with these rules contact andrei@enki.com.

**Comment by enki-hq** [comment]
### Insight

| Insight Name | Error Message |
|:-:|:-:|
| slices | No matching glossary entry found in the database or this pull request. |
| make-a-slice | No matching glossary entry found in the database or this pull request. |
### Workout

| Workout Name | Error Message |
|:-:|:-:|
| extra-types-i | One of the mentioned insights couldn't be found in this pull request or the database. |
### Changelog

| File Name | Error Message |
|:-:|:-:|
| Pull Request | Couldn't find a CHANGELOG entry for this pull request. |

> If you notice anything wrong with these rules contact andrei@enki.com.

**Comment by enki-hq** [comment]
### Changelog

| File Name | Error Message |
|:-:|:-:|
| Pull Request | Couldn't find a CHANGELOG entry for this pull request. |

> If you notice anything wrong with these rules contact andrei@enki.com.

File: go/intro/extra-types-i/README.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+name: Extra Types
```
Comment (nem035) [inline go/intro/extra-types-i/README.md:1]: Would the name "Pointers, Structs, Arrays & Slices" work? 

Also, the description doesn't mention slices. Is that deliberate?

File: go/intro/extra-types-ii/README.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
+name: More on Types
```
Comment (nem035) [inline go/intro/extra-types-ii/README.md:1]: Can we make this name a bit more specific too?

Something like: "Using ranges, slices and maps"? I guess the insight "functions are values" is kind of hard to fit into the title.

I'll leave it up to you to decide what to do 👍

File: go/intro/extra-types-i/README.md (hunk 1 above)
Comment (Stefan-Stojanovic) [reply go/intro/extra-types-i/README.md:1]: It would

**Comment by enki-hq** [comment]
### Course

| Course Name | Error Message |
|:-:|:-:|
| intro | We couldn't find these workouts: `extra-types-i`, `extra-types-ii`. |
### Changelog

| File Name | Error Message |
|:-:|:-:|
| Pull Request | Couldn't find a CHANGELOG entry for this pull request. |

> If you notice anything wrong with these rules contact andrei@enki.com.

**Comment by enki-hq** [comment]
### Changelog

| File Name | Error Message |
|:-:|:-:|
| Pull Request | Couldn't find a CHANGELOG entry for this pull request. |

> If you notice anything wrong with these rules contact andrei@enki.com.

**Comment by enki-hq** [comment]
### Insight previews:
- [ ] [enkidevs/curriculum/go/intro/pointers-structs-arrays-slices/go-arrays.md](https://app.enkipro.com/#/public/preview-insight/enkidevs/curriculum/go/intro/pointers-structs-arrays-slices/go-arrays.md?ref=21db5667a87615ee07c62c53408bc138ee6135bd)
- [ ] [enkidevs/curriculum/go/intro/pointers-structs-arrays-slices/pointers.md](https://app.enkipro.com/#/public/preview-insight/enkidevs/curriculum/go/intro/pointers-structs-arrays-slices/pointers.md?ref=21db5667a87615ee07c62c53408bc138ee6135bd)
- [ ] [enkidevs/curriculum/go/intro/pointers-structs-arrays-slices/slices-ii.md](https://app.enkipro.com/#/public/preview-insight/enkidevs/curriculum/go/intro/pointers-structs-arrays-slices/slices-ii.md?ref=21db5667a87615ee07c62c53408bc138ee6135bd)
- [ ] [enkidevs/curriculum/go/intro/pointers-structs-arrays-slices/slices.md](https://app.enkipro.com/#/public/preview-insight/enkidevs/curriculum/go/intro/pointers-structs-arrays-slices/slices.md?ref=21db5667a87615ee07c62c53408bc138ee6135bd)
- [ ] [enkidevs/curriculum/go/intro/pointers-structs-arrays-slices/structs.md](https://app.enkipro.com/#/public/preview-insight/enkidevs/curriculum/go/intro/pointers-structs-arrays-slices/structs.md?ref=21db5667a87615ee07c62c53408bc138ee6135bd)
- [ ] [enkidevs/curriculum/go/intro/range-map-make/append-to-a-slice.md](https://app.enkipro.com/#/public/preview-insight/enkidevs/curriculum/go/intro/range-map-make/append-to-a-slice.md?ref=21db5667a87615ee07c62c53408bc138ee6135bd)
- [ ] [enkidevs/curriculum/go/intro/range-map-make/functions-are-values.md](https://app.enkipro.com/#/public/preview-insight/enkidevs/curriculum/go/intro/range-map-make/functions-are-values.md?ref=21db5667a87615ee07c62c53408bc138ee6135bd)
- [ ] [enkidevs/curriculum/go/intro/range-map-make/go-range.md](https://app.enkipro.com/#/public/preview-insight/enkidevs/curriculum/go/intro/range-map-make/go-range.md?ref=21db5667a87615ee07c62c53408bc138ee6135bd)
- [ ] [enkidevs/curriculum/go/intro/range-map-make/make-a-slice.md](https://app.enkipro.com/#/public/preview-insight/enkidevs/curriculum/go/intro/range-map-make/make-a-slice.md?ref=21db5667a87615ee07c62c53408bc138ee6135bd)
- [ ] [enkidevs/curriculum/go/intro/range-map-make/map.md](https://app.enkipro.com/#/public/preview-insight/enkidevs/curriculum/go/intro/range-map-make/map.md?ref=21db5667a87615ee07c62c53408bc138ee6135bd)

**Review by kapnobatai137 — APPROVED** [review]

**Review by kapnobatai137 — APPROVED** [review]

**Pull request merged**